templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+
+  <p>Are you sure to delete: {{ car }}</p>
+  <p><i>All Drivers also to be deleted!</i></p>
+
+  <form action="" method="POST">
+  {% csrf_token %}
+  <input type="submit" value="Yes" class="btn btn-danger">
+  </form>
+
+{% endblock %}
```

Comment:
break line at the end of the HTML files

Reply:
![Screenshot_74](https://user-images.githubusercontent.com/110389820/206849556-0220e411-9098-4673-adc7-5e701c2ba02e.png)
![Screenshot_73](https://user-images.githubusercontent.com/110389820/206849558-949f798a-3330-4c4e-a92c-8b85713a81a7.png)